build.gradle

```diff
@@ -2,28 +2,27 @@ import com.modrinth.minotaur.TaskModrinthUpload
 
 plugins {
 	id "java-library"
-	id "fabric-loom" version "0.5-SNAPSHOT"
+	id "fabric-loom" version "0.8-SNAPSHOT"
```

Comment:
According to https://fabricmc.net/versions.html "The recommended loom version for Minecraft 1.17 is 0.9.+", so: 
`id "fabric-loom" version "0.9.+"`.